src/actions/users.js

```diff
@@ -79,15 +79,15 @@ export function createUser(user, data, hash, inviteId) {
     };
 }
 
-export function login(loginId, password, mfaToken = '') {
+export function login(loginId, password, mfaToken = '', ldapOnly = false) {
     return async (dispatch, getState) => {
         dispatch({type: UserTypes.LOGIN_REQUEST}, getState);
 
         const deviceId = getState().entities.general.deviceToken;
 
         let data;
         try {
-            data = await Client4.login(loginId, password, mfaToken, deviceId);
+            data = await Client4.login(loginId, password, mfaToken, deviceId, ldapOnly);
```

Comment:
this is great and all, but how do you decide is `ldapOnly`? asking to see if RN needs to do something to set that value